gh_pages/_source/session5/Building-a-Perception-Pipeline.rst

```diff
@@ -111,16 +113,16 @@ Implement Voxel Filter
        * Fill Code: VOXEL GRID
        * ========================================*/
 
-   Copy and paste the following beneath that banner.
+   Copy and paste the following code beneath that banner.
 
    .. code-block:: c++
 
-      pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_ptr (new pcl::PointCloud<pcl::PointXYZ> (cloud));
-      pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_voxel_filtered (new pcl::PointCloud<pcl::PointXYZ> ());
+      pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_ptr(new pcl::PointCloud<pcl::PointXYZ>(cloud));
+      pcl::PointCloud<pcl::PointXYZ>::Ptr cloud_voxel_filtered(new pcl::PointCloud<pcl::PointXYZ>());
```

Comment:
PCL has a make_shared() function now that can be used instead of `new`, if you like.

https://pointclouds.org/documentation/memory_8h_source.html